src/main/java/com/synopsys/integration/alert/common/descriptor/DescriptorMap.java

```diff
@@ -88,6 +88,8 @@ public Descriptor getDescriptor(final String name) {
         return descriptorMapping.get(name);
     }
 
+    //TODO a lot of these get methods may do better to return optionals
```

Comment:
Yes these methods should return Option.ofNullable(...)

Reply:
We will probably want to do it in another PR as that will touch a lot of different parts of code